ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -264,31 +264,32 @@ const triggerSet: TriggerSet<Data> = {
           return output.combo!({ first: output.northwest!(), second: output.southeast!() });
         }
       },
-      outputStrings: directionOutputStrings,
+      outputStrings: {
+        northeast: Outputs.dirNE,
+        northwest: Outputs.dirNW,
+        southeast: Outputs.dirSE,
+        southwest: Outputs.dirSW,
+        combo: {
+          en: 'Go ${first} / ${second} (knockback)',
+        },
+      },
     },
     {
       id: 'ZodiarkEx Adikia',
       type: 'StartsUsing',
       netRegex: NetRegexes.startsUsing({ id: '63A9', source: 'Zodiark', capture: false }),
       alertText: (data, _matches, output) => {
         ++data.adikiaCounter;
-        switch (data.adikiaCounter) {
-          case 1:
-            return output.text!();
-          case 2:
-            return output.lookforpython!();
-          case 3:
-            return output.text!();
-          default:
-            return output.lookforpython!();
-        }
+        if (data.adikiaCounter % 2 === 1)
+          return output.lookforpython!();
+        return output.text!();
```

Comment:
I'm probably confused here, and this will *work*, but why are we counting them? There are only two, the first with snakes and the second without. Maybe the counter could instead just be a boolean that gets set after the first one?

Reply:
Oh, thanks.  I didn't even think about it and just rewrote the code as-is.